### PR TITLE
Remove ineffective Docker caching in CI

### PR DESCRIPTION
**Pull request by mythmon**
It doesn't work, and it adds about a minute to our builds.

@relud r? (feel free to merge it with the green button if you like it)

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -4,31 +4,16 @@ machine:
...
     - docker inspect -f '{{.Config.Image}}' normandy:build | tee $CIRCLE_ARTIFACTS/docker-image-shasum256.txt
     # get MaxMind GeoIP database
-    - cd ~/cache/ && ~/normandy/bin/download_geolite2.sh
-    # clean up old cached Docker image and save the new one
-    - |
-      mkdir -p ~/cache/docker
-      rm -f ~/cache/docker/image.tar
-      docker save normandy:build > ~/cache/docker/image.tar
-      ls -l ~/cache/docker
+    - ~/normandy/bin/download_geolite2.sh
```
Comment (mythmon) [inline circle.yml]: I don't know why this used to `cd` into the cache directory before. It always downloads to the root of the git repo anyways.

Comment (relud) [inline circle.yml]: will the geolite db be referenced in the correct place with this?

Comment (relud) [inline circle.yml]: if yes, r+

**Comment by relud** [comment]
will merge if build passes

**Pull request merged**